x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate.tsx

```diff
@@ -26,86 +29,452 @@ interface AnalyticsCollectionIntegrateProps {
+
+  const [selectedTab, setSelectedTab] = React.useState<TabKey>('javascriptEmbed');
+
+  const Steps: Record<TabKey, Array<{ title: string; children: React.ReactElement }>> = {
+    javascriptEmbed: [
+      {
+        title: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepOne.title',
+          {
+            defaultMessage: 'Embed onto site',
+          }
+        ),
+        children: (
+          <>
+            <EuiText grow={false}>
+              <p>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepOne.description',
+                  {
+                    defaultMessage:
+                      'Embed the behavioral analytics JavaScript snippet on every page of the website or application you’d like to track.',
+                  }
+                )}
+              </p>
+              <EuiCodeBlock language="html" isCopyable>
+                {`<script src="${webClientSrc}" data-dsn="${analyticsDNSUrl}"></script>`}
+              </EuiCodeBlock>
+            </EuiText>
+          </>
+        ),
+      },
+      {
+        title: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepTwo.title',
+          {
+            defaultMessage: 'Initialise the client',
+          }
+        ),
+        children: (
+          <>
+            <EuiText grow={false}>
+              <p>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepTwo.description',
+                  {
+                    defaultMessage:
+                      'You must initialise the client before you can track events. We recommend initialising just below the script tag.',
+                  }
+                )}
+              </p>
+              <EuiCodeBlock language="html" isCopyable>
+                {'<script type="text/javascript">window.elasticAnalytics.createTracker();</script>'}
+              </EuiCodeBlock>
+            </EuiText>
+          </>
+        ),
+      },
+      {
+        title: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.searchuiEmbed.stepThree.title',
+          {
+            defaultMessage: 'Track individual events',
+          }
+        ),
+        children: (
+          <>
+            <EuiText grow={false}>
+              <p>
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepThree.description"
+                  defaultMessage="Track individual events, like clicks, by calling the trackEvent method. {link}"
+                  values={{
+                    link: (
+                      <EuiLink href="http://www.elastic.co" target="_blank">
```

Comment:
links are to be updated later once documentation is in

Reply:
Would be good to add a data-telemetry-id to the link for tracking purposes

Reply:
Yes! good idea. Added.